### PR TITLE
COSBETA-158: Update copy on contact person page

### PR DESCRIPTION
**Pull request by tom-bell-softwire**
Update page content and make the contact person name a mandatory field.

### CODE REVIEW

File: cosmetics-web/app/views/responsible_persons/account_wizard/contact_person.slim [hunk 1]
```diff
@@ -8,16 +8,19 @@
...
-        | Contact person
+        | Who is the contact person?
       p
-        | A contact person must be able to give information about the cosmetic products to the
-          national poison information service or market surveillance authorities.
+        | A contact person must be able to give information about the cosmetic products when contacted by
+          the National Poison Information Service (NPIS) or market surveillance authorities (for example,
+          Trading Standards officers).
+      p
+        | We will send an email to this person asking them to confirm that they are a contact person.
```
Comment (tom-bell-softwire) [inline cosmetics-web/app/views/responsible_persons/account_wizard/contact_person.slim]: @adrievdluijt – is the content above correct (it was taken from the prototype)?

Comment (adrievdluijt) [reply cosmetics-web/app/views/responsible_persons/account_wizard/contact_person.slim]: Yes, that's fine.

Comment (adrievdluijt) [reply cosmetics-web/app/views/responsible_persons/account_wizard/contact_person.slim]: For below: better would be "We will send a confirmation email to this address"

Comment (tom-bell-softwire) [reply cosmetics-web/app/views/responsible_persons/account_wizard/contact_person.slim]: Thanks Adrie. We're removing the hint below as it seems redundant, given the sentence above.

**Pull request merged**